tscat_gui/__init__.py

```diff
@@ -22,6 +22,22 @@
 from .utils.helper import get_entity_from_uuid_safe
 
 
+class _TrashAlwaysTopOrBottomSortFilterModel(QtCore.QSortFilterProxyModel):
```

Comment:
Not sure it makes sense to describe the behavior in the class name.
CatalogsFilter is enough (it's just a dedicated|custom filter for catalogs after all).